.github/workflows/debian-tests.yml

```
@@ -0,0 +1,42 @@
+# Check if the binaries we have in this repository actually work when "npm i" is run
+name: Install and test the binary
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version:
+#        - '14'
+        - '16'
+#        - '18'
+        os:
+        - 'ubuntu-22.04'
+        container: [ 'node:16.16-buster-slim', 'node:16.16-bullseye-slim' ]
```

Add Debian sid here as well.

---

```suggestion
        container: [ 'node:16.16-buster-slim', 'node:16.16-bullseye-slim', 'debian:unstable-slim' ]
```

For `unstable-sid` we'd need to install Node.js "manually" inside the container.